pkg/operation/botanist/component/hvpa/logging_test.go

```diff
@@ -27,21 +33,45 @@ var _ = Describe("Logging", func() {
 			loggingConfig, err := CentralLoggingConfiguration()
 
 			Expect(err).NotTo(HaveOccurred())
-			Expect(loggingConfig.Parsers).To(Equal(`[PARSER]
-    Name        hvpaParser
-    Format      regex
-    Regex       ^(?<severity>\w)(?<time>\d{4} [^\s]*)\s+(?<pid>\d+)\s+(?<source>[^ \]]+)\] (?<log>.*)$
-    Time_Key    time
-    Time_Format %m%d %H:%M:%S.%L
-`))
+			Expect(loggingConfig.Filters).To(Equal(
+				[]*fluentresources.ClusterFilter{
+					{
+						ObjectMeta: metav1.ObjectMeta{
+							Name:   DeploymentName,
+							Labels: map[string]string{v1beta1constants.LabelKeyCustomLoggingResource: v1beta1constants.LabelValueCustomLoggingResource},
+						},
+						Spec: fluentresources.FilterSpec{
+							Match: "kubernetes.*" + DeploymentName + "*" + ContainerName + "*",
+							FilterItems: []fluentresources.FilterItem{
+								{
+									Parser: &filter.Parser{
+										KeyName:     "log",
+										Parser:      ContainerName + "-parser",
+										ReserveData: pointer.BoolPtr(true),
```

Comment:
```suggestion
										ReserveData: pointer.Bool(true),
```

You could fix every occurrence of this.